site/docs/webrtc/voice-iw.md

```diff
@@ -12,43 +12,43 @@ image: ../../static/img/bandwidth-logo.png
 
 ## Connecting WebRTC to Programmable Voice
 
-Bandwidth has a demonstrated history of excellence in IP voice technologies, including comprehensive CPaaS Programmable Voice and Messaging capabilities. The introduction of WebRTC provides the opportunity to combine the strength and diversity of our Programmable Voice with our highly controllable WebRTC to yield a rich and flexible platform that allows our customers to extend their business with a broad range of communication capabilities.
+Bandwidth has a demonstrated history of excellence in IP voice technologies, including comprehensive CPaaS Programmable Voice (PV) and Messaging capabilities. The introduction of WebRTC provides the opportunity to combine the strength and diversity of our Programmable Voice (PV) offering with our highly controllable WebRTC to yield a rich and flexible platform that allows our customers to extend their business with a broad range of browser-based communication capabilities.
 
-One of the steps we have taken to enable this capability suite is to ensure that our Programmable Voice (PV) and WebRTC APIs combine their respective strengths when integrated.
+One of the steps we have taken to enable this capability suite is to ensure that our PV and WebRTC APIs combine their respective strengths when integrated.
 
-When PV calls interwork with WebRTC communications, control of the behavior of the voice call is managed by the extensive suite of PV capabilities. Bandwidth offers “BXML” scripts to control call events using action verbs. This gives you the ability to create flexible solutions that enable your application to create voice calls, receive voice calls, and control and monitor changes made to these calls.
+When PV calls interwork with WebRTC communications, the behavior of the voice call can be augmented by the extensive suite of PV capabilities. This enables your application to create voice calls, receive voice calls, insert services like recording, and control and monitor changes made to these calls. This is done with the use of "BXML" verbs supported by the PV platform.
 
-And by combining our WebRTC and programmable voice APIs, you can create composite call flows that meet your needs.
+By combining our WebRTC and programmable voice APIs, you can create composite call flows that meet your needs.
 
-The interaction between PV and WebRTC is initiated under application control from the PV side of the platform.
+To create a Programmable Voice call that involves WebRTC endpoints, your appliction establishes a connection from the Programmable Voice network to the WebRTC service.
```

Comment:
mispelling in application. Couple minor suggestions:

To create a Programmable Voice call that involves WebRTC endpoints, your application **_first_** establishes a connection from the Programmable Voice network to the WebRTC **_platform_**.